templates/file_crash_bug_description.md.jinja

```diff
@@ -0,0 +1,54 @@
+{#
+    Part of this template was extracted from Socorro's repository:
+    https://github.com/mozilla-services/socorro/blob/ff8f5d6b41689e34a6b800577d8ffe383e1e62eb/webapp/crashstats/crashstats/jinja2/crashstats/bug_comment.txt
+    https://github.com/mozilla-services/socorro/blob/ff8f5d6b41689e34a6b800577d8ffe383e1e62eb/webapp/crashstats/signature/jinja2/signature/signature_summary.html#L7-L41
+#}
+Crash report: https://crash-stats.mozilla.org/report/index/{{ uuid }}
+
+{% if java_stack_trace %}
+Java stack trace:
+```
+{{ java_stack_trace|truncate(5000, True)|safe }}
+```
+{% elif frames %}
+{% if moz_crash_reason %}
+MOZ_CRASH Reason: ```{{ moz_crash_reason|safe }}```
+{% elif reason %}
+Reason: ```{{ reason|safe }}```
+{% endif %}
+{% if crashing_thread is none %}
+No crashing thread identified; using thread 0.
+{% endif %}
+Top {{ frames|length }} frames of crashing thread:
+```
+{% for frame in frames -%}
+{{ frame.frame|safe}}  {{ frame.module|safe }}  {{ frame.signature|safe }}  {{ frame.source|safe }}
+{% endfor -%}
+```
+{% endif %}
+
+
+Here are some insights about the crash signature based on recent data at the time of reporting the bug:
+
+- **Crash kind:** {{ "Plugin Crash" if signature.is_plugin_crash  else  "Browser Crash" }}
+- **Is startup crash:** {{ "Yes - " if signature.is_startup_related_crash else "No" }}
```

Comment:
Do you think we should we mention it in the bug title too? This would make it obvious that it is a startup crash even when you look at a list of bugs for example.

Reply:
Yes, it is a good idea. @Archaeopteryx suggested the same. Done in 727d61c76d33b3a00cb8794df494e2aa4be0dbd5.